### PR TITLE
Search SPIRE + LITE

### PR DESCRIPTION
**Pull request by richtier**

### CODE REVIEW

File: caseworker/templates/search/search.html [hunk 1]
```diff
@@ -47,20 +47,30 @@ <h1 class="govuk-heading-l">Search Cases</h1>
                     {% for good in item.goods %}
                     <tr class="govuk-table__row">
                         {% if forloop.counter0 == 0 %}
-                        <td class="govuk-table__cell govuk-table__cell--tight" rowspan="{{ item.goods|length }}">
-                            {% if item.queues %}
+                        <td class="govuk-table__cell govuk-table__cell" rowspan="{{ item.goods|length }}">
+                            {% if item.index == 'lite' %}
+                                {% if item.queues %}
+                                    <a class="govuk-link govuk-link--no-visited-state" id="case-${item.id}"
+                                        href="/queues/{{ item.queues.0.id }}/cases/{{ item.id }}/">
```
Comment (eadpearce) [inline caseworker/templates/search/search.html:54]: does this also want updating to this? `{% url 'spire:application-detail' id=item.id %}`

File: caseworker/templates/search/search.html [hunk 2]
```diff
@@ -102,29 +112,39 @@ <h1 class="govuk-heading-l">Search Cases</h1>
                     </tr>
                     {% empty %}
                     <tr class="govuk-table__row">
-                        <td class="govuk-table__cell govuk-table__cell--tight">
-                            {% if item.queues %}
+                        <td class="govuk-table__cell govuk-table__cell">
+                            {% if item.index == 'lite' %}
+                                {% if item.queues %}
+                                    <a class="govuk-link govuk-link--no-visited-state" id="case-${item.id}"
+                                        href="/queues/{{ item.queues.0.id }}/cases/{{ item.id }}/">
```
Comment (eadpearce) [inline caseworker/templates/search/search.html:119]: same here

File: caseworker/templates/search/search.html (hunk 1 above)
Comment (richtier) [reply caseworker/templates/search/search.html:54]: this is linking to a different place, but sure I'll update to `{% url 'case:name' queue_pk=item.queues.0.id pk=item.id}`

**Review by saruniitr — APPROVED** [review]
one minor change requested, otherwise looks good to me.

**Pull request merged**